aws/rust-runtime/aws-sigv4/src/http_request/settings.rs

```diff
@@ -3,12 +3,14 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-use http::header::{HeaderName, USER_AGENT};
+use http::header::{HeaderName, AUTHORIZATION, USER_AGENT};
 use std::time::Duration;
 
 /// HTTP signing parameters
 pub type SigningParams<'a> = crate::SigningParams<'a, SigningSettings>;
 
+const X_RAY_TRACE_HEADER: &str = "x-amzn-trace-id";
```

Comment:
[`HeaderName::from_static`](https://docs.rs/http/0.2.9/http/header/struct.HeaderName.html#method.from_static) is `const`, so you can actually put the entire `HeaderName` in this constant rather than creating it every time signing settings is created.

Reply:
👍 I got caught up trying to make the array static